stubs/aws-xray-sdk/aws_xray_sdk/core/emitters/udp_emitter.pyi

```diff
@@ -10,10 +11,12 @@ PROTOCOL_DELIMITER: str
 DEFAULT_DAEMON_ADDRESS: str
 
 class UDPEmitter:
-    def __init__(self, daemon_address="127.0.0.1:2000") -> None: ...
-    def send_entity(self, entity) -> None: ...
-    def set_daemon_address(self, address) -> None: ...
+    def __init__(self, daemon_address: str = "127.0.0.1:2000") -> None: ...
+    def send_entity(self, entity: Any) -> None: ...
```

Comment:
We require a comment (or type alias) for any use of `Any`. In this case, we could do better by adding a protocol like this and using it here:

```python
@type_check_only
class _HasSerialize(Protocol):
    def serialize(self) -> str: ...
```